gpu-service/jobs.py

```
@@ -0,0 +1,288 @@
+            tmp_image.write(image_bytes)
+            tmp_image.flush()
+
+            image = face_recognition.load_image_file(tmp_image.name)
+            faces = RetinaFace.detect_faces(tmp_image.name)
+
+            if not isinstance(faces, dict):
+                logger.info(f"No faces detected in {object_name}")
+                return {"status": "success", "message": "No faces detected"}
+
+            points_to_upsert = []
+            for idx, (face_key, face_data) in enumerate(faces.items()):
```

Ruff (0.11.9)

Loop control variable `face_key` not used within loop body

Rename unused `face_key` to `_face_key`

(B007)